static/app/components/idBadge/projectBadge.tsx

```diff
@@ -59,6 +59,7 @@ const ProjectBadge = ({
 
 const StyledLink = styled(Link)`
   flex-shrink: 0;
+  width: 100%;
```

Comment:
Fixes the column issue with the ProjectLink. Might cause issues elsewhere, so I need to double check the snapshots.

Reply:
i fixed in https://github.com/getsentry/sentry/pull/25506

Reply:
```suggestion
```